### ISSUE TITLE
ImageNet-VID dataset

### ISSUE THREAD

**Issue by Actmaiji** [HyeonseobNam/py-MDNet#37]
hi,what do you mean the "ImageNet-VID dataset" ? Because I can find the 2015  :http://bvisionweb1.cs.unc.edu/ilsvrc2015/download-videos-3j16.php#vid  and the 2017.
Can anyone give me a link?thanks a lot.

**Comment by hyeonseobnam** [comment]
Hi, @Actmaiji 
It is the 2015 version. I will add the link that you mentioned to the readme. Thank you :)

**Comment by Actmaiji** [comment]
thanks a lot.But I still have two  quetions:first,when you training on vot ot ImageNet-VID,do you have any method to verify the training results(two classification model) or just verify on online tracking? during training ,the precision reach 0.994(Imagenet),but the online tracking results is worse  than 0.971.thanks a lot

**Comment by hyeonseobnam** [comment]
@Actmaiji I verified the pretraining policy using the last 20% of frames for each video as a separated validation set, and add them to the training set for the final release. The online finetuning is just verified on tracking.

**Comment by Actmaiji** [comment]
Thanks a lots for your replying.Sorry,I coundn't understand what the "trainging set" and the "final release" exactly mean.Do you mean that when n_cycle=50. In  # Main trainig loop when i = 49,we added the validation set to the training set, and the model parameters still update? And the final release  is the final model?Thank you :)

**Comment by hyeonseobnam** [comment]
The "training set" means the entire data used for pretraining.
To fine appropriate hyperparameters (learning rate, number of cycles, etc.) for pretraining, I used 80% of Imagenet-VID as a training set and the rest 20% as a validation set. 
After finding the hyperparameters, I re-train the model from the scratch using the entire Imagenet-VID as a training set without a separated validation set, which is what the "final release" of the code (pretrain/train_mdnet.py) does.

**Comment by Actmaiji** [comment]
Thank you. I got it.Did you use the same training strategy on vot training dataset?  I used to verify it on
 tracking the data set, just initializing the first frame and then testing the two classification model.Do you think that will work?

**Comment by sieumap43** [comment]
> hi,what do you mean the "ImageNet-VID dataset" ? Because I can find the 2015 :http://bvisionweb1.cs.unc.edu/ilsvrc2015/download-videos-3j16.php#vid and the 2017. Can any guy give me a link of the ImageNet-VID dataset ?thanks a lot.

The link just died recently. Does someone have the dataset or an alternative link?
